Core.ApplicationServices/Organizations/OrganizationRightsService.cs

```diff
@@ -75,6 +85,64 @@ public class OrganizationRightsService : IOrganizationRightsService
             return RemoveRight(right);
         }
 
+        public Maybe<OperationError> RemoveUnitRole(int rightId)
```

Comment:
rename this to removeorganizationunitright... you are not providing a role but a right id